.idea/runConfigurations/Winery_Server.xml

```diff
@@ -49,6 +52,36 @@
         <option name="VM_PARAMETERS" value="" />
         <option name="PROGRAM_PARAMETERS" value="" />
       </SHUTDOWN>
+      <EnvironmentVariables>
```

Comment:
Is this required to push into the default config?
... Maybe a separate would be better.